tests/e2e/standard/wdio.conf.js

```diff
@@ -9,9 +9,11 @@ chai.use(require('chai-exclude'));
 
 // Override specific properties from wdio base config
 const standardConfig = Object.assign(wdioBaseConfig.config, {
-  specs: [
-    '**/*.wdio-spec.js'
-  ],
+  suites: {
+    all: [
+      './**/*.wdio-spec.js'
+    ]
+  },
```

Comment:
Maybe we can start defining suites also for the standard config, thinking that we probably will add more tests in the feature.

I think that the suites that we create for `default` should be also for `standard`

Reply:
Definitely, if other tests grow to be a kind of bottleneck as the default tests were becoming, we should think of splitting them too.